web/app/scripts/resources/query-builder-service.js

```diff
@@ -42,9 +47,10 @@
          *
          * @param {bool} doFilter If true: filter results
          */
-        function windshaftQuery(doFilter) {
+        function windshaftQuery(extraParams, doFilter) {
```

Comment:
It'd be good to make sure the placement of `doFilter` in the parameter list is the same across all functions that use it.